server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/ProjectDashboardPage.java

```diff
@@ -17,80 +17,77 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonarqube.pageobjects;
+package org.sonarqube.qa.util.pageobjects;
 
+import com.codeborne.selenide.Condition;
 import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.Selenide;
 import com.codeborne.selenide.SelenideElement;
 import java.util.Arrays;
 import org.openqa.selenium.By;
 
-import static com.codeborne.selenide.Condition.exist;
-import static com.codeborne.selenide.Condition.hasText;
-import static com.codeborne.selenide.Condition.text;
-import static com.codeborne.selenide.Condition.visible;
-import static com.codeborne.selenide.Selenide.$;
-import static com.codeborne.selenide.Selenide.$$;
 import static org.assertj.core.api.Assertions.assertThat;
 
+
 public class ProjectDashboardPage {
 
   public ProjectDashboardPage() {
-    $(".overview").shouldBe(visible);
+    Selenide.$(".overview").shouldBe(Condition.visible);
   }
 
   public SelenideElement getLinesOfCode() {
-    SelenideElement element = $("#overview-ncloc");
-    element.shouldBe(visible);
+    SelenideElement element = Selenide.$("#overview-ncloc");
+    element.shouldBe(Condition.visible);
     return element;
   }
 
   public SelenideElement getLanguageDistribution() {
-    SelenideElement element = $("#overview-language-distribution");
-    element.shouldBe(visible);
+    SelenideElement element = Selenide.$("#overview-language-distribution");
+    element.shouldBe(Condition.visible);
     return element;
   }
 
   public SelenideElement getOverviewMeasure(String measure) {
-    ElementsCollection measures = $$(".overview-domain-measure");
-    SelenideElement element = measures.find(text(measure)).shouldBe(visible);
+    ElementsCollection measures = Selenide.$$(".overview-domain-measure");
+    SelenideElement element = measures.find(Condition.text(measure)).shouldBe(Condition.visible);
     return element;
   }
 
   private SelenideElement getTagsMeta() {
-    SelenideElement element = $(".overview-meta-tags");
-    element.shouldBe(visible);
+    SelenideElement element = Selenide.$(".overview-meta-tags");
+    element.shouldBe(Condition.visible);
     return element;
   }
 
   public ProjectDashboardPage shouldHaveTags(String... tags) {
     String tagsList = String.join(", ", Arrays.asList(tags));
-    this.getTagsMeta().$(".tags-list > span").should(hasText(tagsList));
+    this.getTagsMeta().$(".tags-list > span").should(Condition.hasText(tagsList));
     return this;
   }
 
   public ProjectDashboardPage shouldNotBeEditable() {
     SelenideElement tagsElem = this.getTagsMeta();
-    tagsElem.$("button").shouldNot(exist);
-    tagsElem.$("div.multi-select").shouldNot(exist);
+    tagsElem.$("button").shouldNot(Condition.exist);
+    tagsElem.$("div.multi-select").shouldNot(Condition.exist);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "div.multi-select" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)